src/test/java/se/nackademin/LemurTest.java

```diff
@@ -0,0 +1,72 @@
+package se.nackademin;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class LemurTest {
+    @Test
+    public void shouldEat() {
```

Comment:
- Your tests shouldEat, shouldNotEat etc contains a number of tests based that depends on some comparison, e.g. that a value should be higher than or lower than a constant. You should document this comparison & constant, best would be with comments in the test. E.g. how do we tell other developers that 3 should be considered not hungry and 8 should be considered hungry, and where the limit is?
- It would make sense to test edge cases, e.g. to test 5 - should that be counted as hungry or not hungry? What about -2 ?